pkg/utils/taints.go

```diff
@@ -79,7 +84,9 @@ func setOutOfTaintSupportedFlag(version *version.Info) error {
 		return err
 	}
 
-	IsOutOfServiceTaintSupported = majorVer > minK8sMajorVersionSupportingOutOfServiceTaint || (majorVer == minK8sMajorVersionSupportingOutOfServiceTaint && minorVer >= minK8sMinorVersionSupportingOutOfServiceTaint)
+	IsOutOfServiceTaintSupported = majorVer > minK8sMajorVersionOutOfServiceTaint || (majorVer == minK8sMajorVersionOutOfServiceTaint && minorVer >= minK8sMinorVersionSupportingOutOfServiceTaint)
 	utilTaintsLog.Info("out of service taint strategy", "isSupported", IsOutOfServiceTaintSupported, "k8sMajorVersion", majorVer, "k8sMinorVersion", minorVer)
+	IsOutOfServiceTaintGA = majorVer > minK8sMajorVersionOutOfServiceTaint || (majorVer == minK8sMajorVersionOutOfServiceTaint && minorVer >= minK8sMinorVersionGAOutOfServiceTaint)
+	utilTaintsLog.Info("out of service taint strategy", "isGA", IsOutOfServiceTaintGA, "k8sMajorVersion", majorVer, "k8sMinorVersion", minorVer)
```

Comment:
IIUC for k8s version 1.28+ SNR would have two prints, one for that out-of-service taint being supported and one for being GA. Can we maybe only print that it is GA in the case of k8s version 1.28+ and when the k8s version is  either 1.27/1.26, then we would only print that out-of-service taint is supported?

Reply:
Hmm, I think I'm leaning into keeping as is, it seems like a bit of overcomplicating it without a real benefit.
Keep in mind that this log is only printed only once when the operator starts.

Reply:
SGTM